src/sink.rs

```diff
@@ -15,6 +15,9 @@ use crate::Source;
 ///
 /// Dropping the `Sink` stops all sounds. You can use `detach` if you want the sounds to continue
 /// playing.
+///
+/// Once the sink has finished playing all sounds in its queue, the handle stops looking for more sounds, and so the 
+/// `Sink` will stop working. Create a new `Sink` to continue playing sounds.
```

Comment:
```suggestion
/// Once the sink has finished playing all sounds in its queue, it will stop. Call the `append` method
/// to provide it with additional sounds to play.
```